Error: Linutil Updater failed to compile `linutil_tui` v24.9.19

## Describe the bug
The Linutil Updater tool fails due to a failed compilation of the `linutil_tui` dependency.

## To Reproduce
Steps to reproduce the behavior:
1. Attempt to run Linutil Updater tool (on Debian Testing Branch) from the "Applications setup" collection
2. Observe the compilation error

## Expected behavior
The Linutil Updater should compile and run without errors, allowing the user to update applications as intended.

## Error message
```bach
error[E0164]: expected tuple struct or tuple variant, found struct variant `Command::LocalFile`
  --> /home/ismail/.cargo/registry/src/index.crates.io-6f17d22bba15001f/linutil_tui-24.9.19/src/floating_text.rs:40:13
   |
40 |             Command::LocalFile(file_path) => {
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not a tuple struct or tuple variant

error[E0164]: expected tuple struct or tuple variant, found struct variant `Command::LocalFile`
   --> /home/ismail/.cargo/registry/src/index.crates.io-6f17d22bba15001f/linutil_tui-24.9.19/src/running_command.rs:151:17
    |
151 |                 Command::LocalFile(file) => {
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^ not a tuple struct or tuple variant
For more information about this error, try `rustc --explain E0164`.
error: could not compile `linutil_tui` (bin "linutil") due to 2 previous errors
error: failed to compile `linutil_tui v24.9.19`, intermediate artifacts can be found at `/tmp/cargo-installijJJVW`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```
## Screenshots
![image](https://github.com/user-attachments/assets/6d0ae4fb-9ef5-42d7-9e39-439bb6ed2d78)

## Additional context
- Linutil version: Pre-Release 2024.09.28
- OS: Debian 12 Testing Branch
- Kernel version: Linux 6.10.9-amd64
- Rust version: rustc 1.81.0 (eeb90cda1 2024-09-04)

## Checklist
- [x] I checked for duplicate issues.
- [x] I checked already existing discussions.
- [x] This issue is not included in the roadmap.
- [ ] This issue is present on both stable and development branches.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [workspace.package]
 license = "MIT"
-version = "24.9.23"
+version = "24.9.28"
 
 [workspace.dependencies]
 ego-tree = "0.6.2"

--- a/tui/Cargo.toml
+++ b/tui/Cargo.toml
@@ -24,7 +24,7 @@ ratatui = "0.28.1"
 tui-term = "0.1.12"
 unicode-width = "0.1.13"
 rand = { version = "0.8.5", optional = true }
-linutil_core = { path = "../core", version = "24.9.23" }
+linutil_core = { path = "../core", version = "24.9.28" }
 tree-sitter-highlight = "0.23.0"
 tree-sitter-bash = "0.23.1"
 anstyle = "1.0.8"

